src/Reflection/ReflectionProperty.php

```diff
@@ -38,6 +38,16 @@ class ReflectionProperty implements \Reflector
      */
     private $declaringClass;
 
+    /**
+     * @var string
+     */
+    private $docBlock = '';
+
+    /**
+     * @var bool
+     */
+    private $isRuntimeDeclared = false;
```

Comment:
Yes, it's true, but it makes intent much clearer when reading the code in the `isDefault()` implementation.